src/deadline/houdini_submitter/otls/deadline_cloud.hda/Driver_1deadline__cloud/OnCreated

```diff
@@ -1,16 +1,7 @@
 from botocore.exceptions import CredentialRetrievalError
-from deadline.client.api import get_boto3_client
-from deadline.client.config import get_setting
 
 try:
     node = kwargs['node']
-    deadline = get_boto3_client('deadline')
-    farm_id = get_setting('defaults.farm_id')
-    farm_response = deadline.get_farm(farmId=farm_id)
-    node.parm('farm').set(farm_response['displayName'])
-    queue_id = get_setting('defaults.queue_id')
-    queue_response = deadline.get_queue(farmId=farm_id, queueId=queue_id)
-    node.parm('queue').set(queue_response['displayName'])
```

Comment:
The result of this change means that on node creation we won't populate the farm and queue, even though they may be set in the deadline config already (managed outside of houdini)?

Can we add the check here? Or is it handled elsewhere?

Reply:
This logic is now included in the `update_queue_parameters_callback` function!

Reply:
So it is! Turns out `update_queue_parameters_callback` != `update_queue_parameters`. Thanks!